packages/falcon-magento2-api/src/index.js

```diff
@@ -65,6 +66,31 @@ module.exports = class Magento2Api extends Magento2ApiBase {
     return currencies;
   }
 
+  /**
+   * Fetch Menu
+   * @param {object} obj Parent object
+   * @param {object} params - request params
+   * @return {Promise<MenuItem[]>} requested Menu data
+   */
+  async menu() {
+    const response = await this.get('/menu');
+    const { data } = this.convertKeys(response);
+
+    const mapMenu = x => {
+      if (Array.isArray(x)) {
+        return x.length > 0 ? x.map(mapMenu) : [];
+      }
+
+      return {
+        ...x,
+        urlPath: urlJoin(x.urlPath, undefined, { leadingSlash: true }),
```

Comment:
@tutok we could simply add a `/` prefix depending on its value (without using a dedicated package).

```suggestion
        urlPath: x.urlPath.startsWith('/') ? x.urlPath : `/${x.urlPath}`,
```

Reply:
and what in a situation when we will have urlPath like `//foo/bar`?

Reply:
@tutok if `/` is already there - we won't be adding it. If there are 2 `/` there - we shouldn't be fixing it (maybe there's such intention behind it)

Reply:
I decided to use external package because I feel that I can not assume what kind of url exacly Magento can return. While we need to normalize these to be able to use them in falcon-client. So instead of writeing code myself to recognize all use cases
- if leadingSlash exists
- if protocol exists
- is trailingSlash exists
found that `proper-url-join` cover all of them 

```
urlJoin('https://www.google.com/', undefined, { leadingSlash: true })
> "https://www.google.com"

urlJoin('www.google.com/', undefined, { leadingSlash: true })
> "/www.google.com"

urlJoin('//www.google.com/', undefined, { leadingSlash: true })
> "/www.google.com"

urlJoin('www.google.com//', undefined, { leadingSlash: true })
> "/www.google.com"

urlJoin('www.google.com', undefined, { leadingSlash: true })
> "/www.google.com"

```